### ISSUE TITLE
Bibtex rendering of history citations tweaks

### ISSUE THREAD

**Issue by dannon** [galaxyproject/galaxy#15477]
Discussed in UI/UX, text in bibtex mode should be copyable (with a clipboard button?), should not include links to copy out.

![image](https://user-images.githubusercontent.com/155398/216089640-315a3127-998a-4294-b458-8b0c965aad7b.png)

**Comment by dadrasarmin** [comment]
Is it possible to have a download button (similar to copy button)? Then, researchers can download citations in BibTex, EndNote, RefMan, or Reworks formats and import it into their citation managers. 

At the moment, the process requires copy paste and manually saving the file.

**Comment by Sch-Da** [comment]
I agree with @dadrasarmin, the download option would be very helpful!

**Comment by dannon** [comment]
See #20402 for some improvements here that should cover the requested functionality.  Thanks for bumping this one!

**Comment by dadrasarmin** [comment]
Thanks a lot. This is really useful.

**Comment by Sch-Da** [comment]
Thanks a lot, both of you, for making this happen!
